apistructs/org.go

```diff
@@ -75,7 +75,8 @@ type OrgSearchRequest struct {
 	PageNo   int `query:"pageNo"`
 	PageSize int `query:"pageSize"`
 
-	OrgID string
+	OrgID string `query:"orgId"`
+	Org   string `query:"org"`
```

Comment:
Add comment for these two similar variables, describe the reason why add new variable `org`.